crates/turborepo-lib/src/cli.rs

```diff
@@ -325,8 +325,8 @@ pub struct RunArgs {
     #[clap(short = 'F', long, action = ArgAction::Append)]
     pub filter: Vec<String>,
     /// Ignore the existing cache (to force execution)
-    #[clap(long)]
-    pub force: bool,
+    #[clap(long, env = "TURBO_FORCE", default_missing_value = "false")]
```

Comment:
I'm guessing missing value here should be true

Reply:
@mehulkar Ah, thanks for catching it! Fixed 👍